packages/server/core/src/base/middlewares/customServer.ts

```diff
@@ -90,13 +90,13 @@ export class CustomServer {
         }
       }
 
+      await next();
```

Comment:
Why call next earlier than c.finalized `return`

Reply:
In the next, some middlewares may also want to end the response, such as the ssr handler for redirect.